src/System.Runtime/tests/System/Reflection/InvokeRefReturn.netcoreapp.cs

```diff
@@ -0,0 +1,152 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+
+using Xunit;
+
+namespace System.Reflection.Tests
+{
+    public class InvokeRefReturnNetcoreTests
+    {
+        [Theory]
+        [MemberData(nameof(RefReturnInvokeTestData))]
+        [ActiveIssue("TFS 603305 - Bring Project N up to sync", TargetFrameworkMonikers.UapAot)]
+        public static void TestRefReturnPropertyGetValue<T>(T value)
```

Comment:
This is a generic method that I assume xUnit instantiates using `MakeGenericMethod`. This will only test universal shared  code and the slow calling convention interpreter on .NET Native.

But I guess that's .NET Native's problem to test. We just need to make sure we don't forget to fork/update these tests into .NET Native's test collateral.